pxtblocks/blocklyloader.ts

```diff
@@ -612,7 +649,7 @@ namespace pxt.blocks {
                 if (!this.isCollapsed()) {
                     let option: any = { enabled: true };
                     let name = this.getFieldValue('VAR');
-                    option.text = Blockly.Msg.VARIABLES_SET_CREATE_GET.replace('%1', name);
+                    option.text = lf("Create 'get %1'").replace('%1', name);
```

Comment:
``lf`` supports arguments which will be substitued to {0}, {1}, ...

   lf("Create 'get {0}', name)

See example at https://github.com/Microsoft/pxt/blob/c3f64ddcd6533ff8338346d12a7e291b03c7a7f1/webapp/src/appcache.ts#L13